src/solution_part2.kt

```diff
@@ -1,3 +1,15 @@
 fun main(){
- print("I love Kotlin!!!!")
+ val  playerOne=90
+ val   playerTwo=30
+ if (playerOne <= 90){
+  if (playerTwo<=30){

```

Comment:
wrong logic 
 
  val playerOne = 79
 when(playerOne){
    in 50..89 -> println("the player's health is fine")
    in 0..49 -> println("the player's health is awful")
   else -> println("Score not valid")
  }